src/phases/scale/LargeScalePhases.yml

```diff
@@ -235,6 +236,25 @@ SnapshotScanner5GigabytesCmd:
   CollectionSortOrder: forward
   GlobalRate: 1 per 5 minutes
 
+FixedRateSnapshotScanner5GigabytesCmd:
+  Duration: *Duration
+  ScanSizeBytes: 5000000000
+  # tetsuo-cpp: We don't supply scan duration since documents and scan rate implies it.
+  #
+  # This is how it's described in the technical design but I'm thinking that scan rate
+  # should be bytes per second rather than documents per second.
+  #
+  # The reasoning is that we don't know the document size. So for example, we could scan
+  # 100 large documents and exceed the 5GB size meaning that the scan will complete within
+  # a second. Similarly, we could scan 100000 tiny documents and finish the phase without
+  # hitting 5GB in total.
+  Documents: 100000
+  ScanRate: 100 per 1 second
```

Comment:
My understanding is that this isn't supported out of the box so we'll have to tweak the underlying actor to get this to work.

Reply:
@ddanderson 
From the technical design:

> Our modified scan will have a fixed scan rate and a fixed total number of operations to complete.  Thus, provided it can keep up, it will run for a fixed amount of time.  As a strawman example, let’s propose 100 reads per second, and 100,000 total reads.  So the scan will finish in 1000 seconds.

Am I right in thinking that the `CollectionScanner` actor can't be configured to do this and requires some changes to the underlying C++? As of now, I've scheduled some time to implement this functionality in WT-6966.

Reply:
@tetsuo-cpp You're right.  There's a way to set the total number of bytes to scan, but I don't see a way to limit the rate.

Reply:
Ok, we'll address this in WT-6966.